docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+# Commissioning nRF Connect Accessory using Android CHIPTool
+
+This article describes how to use [CHIPTool](../../src/android/CHIPTool/README.md) for Android smartphones to commission a Nordic Semiconductor nRF52840 DK running
+[nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md) onto a CHIP-enabled Thread network. The instructions are also valid for [nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md).
+
+Prerequisites:
+- 2x nRF52840 DK (PCA10056)
+  > :information_source: One nRF52840 DK can be replaced with another [OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor) compatible device like nRF52840 Dongle
+- 1x smartphone with Android 8+
+- 1x PC with Ubuntu 20.04 and Docker installed, with a spare Wi-Fi card.<br>
+  > :information_source: While the article uses Ubuntu 20.04 as a reference, all the presented steps should be achievable using other popular operating systems.
```

Comment:
I would move these prerequisites after the diagram and before Building and flashing.
I think having an idea of how the commissioning works makes it easier to understand why I need these prerequisites, not the other way around.